rust/projects/parallelism/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "kvs"
+version = "0.1.0"
+authors = ["Brian Anderson <andersrb@gmail.com>"]
+description = "A key-value store"
+edition = "2018"
+
+[dependencies]
+clap = "2.33.0"
+structopt = "0.2.15"
+failure = "0.1.5"
+serde = { version = "1.0.89", features = ["derive"] }
+serde_json = "1.0.39"
+log = "0.4.6"
+env_logger = "0.6.1"
+sled = "0.22.1"
+crossbeam = "0.7.1"
+rayon = "1.0.3"
+num_cpus = "1.10.0"
+crossbeam-skiplist = { version = "0.0.0", git = "https://github.com/crossbeam-rs/crossbeam.git", rev = "8cc906b" }
```

Comment:
Needing to use a git dep is actually a good opportunity for some nuanced practical discussion. Nice.